ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +35,209 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  function  assertAddTermCommon(assert, component) {
+    /** Common tests for Add terms, use in term and taxonomy components */
```

Comment:
These are not tests, they are assertions. There is nothing being changed in this function, just values being asserted. Can you change the JSDoc accordingly? Also place this above the function definition and add information about parameters (see below comment).